public/app/features/profile/ChangePasswordPage.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { PureComponent } from 'react';
+import { hot } from 'react-hot-loader';
+import { connect } from 'react-redux';
+import { StoreState } from 'app/types';
+import { NavModel } from '@grafana/data';
+import { getNavModel } from 'app/core/selectors/navModel';
+import { UserProvider } from 'app/core/utils/UserProvider';
+import Page from 'app/core/components/Page/Page';
+import { ChangePasswordForm } from './ChangePasswordForm';
+
+export interface Props {
+  navModel: NavModel;
+}
+
+export class ChangePasswordPage extends PureComponent<Props> {
+  render() {
+    const { navModel } = this.props;
+    const isLoading = false;
+    return (
+      <Page navModel={navModel}>
+        <UserProvider>
+          {({ changePassword }, states) => (
+            <Page.Contents isLoading={isLoading}>
```

Comment:
What's the point of `isLoading = false` here?

Reply:
For some reason, I thought this prop wasn't optional but it is. Thanks for that!